airbyte-integrations/connectors/source-mailersend/source_mailersend/spec.yaml

```diff
@@ -0,0 +1,29 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Mailersend Spec
+  type: object
+  required:
+    - api_token
+    - domain_id
+  additionalProperties: true
+  properties:
+    api_token:
+      type: string
+      description: Your API Token. See <a href="https://www.mailersend.com/help/managing-api-tokens">here</a>.
+    domain_id:
+      type: string
+      description: The domain entity in mailersend
+      examples:
+        - airbyte.com
+        - linkana.com
+    date_from:
+      type: string
+      description: Timestamp is assumed to be UTC. Must be lower than date_to.
+      examples:
+        - "1443651141"
+    date_to:
+      type: string
+      description: Timestamp is assumed to be UTC. Must be higher than date_from.
+      examples:
+        - "1443651141"
```

Comment:
Can you rename this to start_date and end_date?

Reply:
@marcosmarxm, date_from and date_to is how mailersend named it on [docs](https://developers.mailersend.com/api/v1/activity.html#get-a-list-of-activities)

<img width="1014" alt="Screenshot 2022-10-31 at 7 43 57 PM" src="https://user-images.githubusercontent.com/727781/199123810-8ffb369e-4d67-4c3b-bfa7-4e52f7bcb4a6.png">

Should I renamed it?

Reply:
I think we can rename because the user don't need to know the name the API uses, probably is better to use a more common terminology inside Airbyte.

Reply:
got it!